templates/kernel/lifecycle.html

```diff
@@ -0,0 +1,747 @@
+          <p>The 16.04.2 and newer point releases ship with an updated kernel and X stack by default for the desktop. Server installations default to the GA kernel and provide the enablement kernel as optional.The 16.04 HWE Stacks follow the Rolling Update Model.</p>
+        </div>
+
+        <div class="p-tabs__content" id="installation-18-04" role="tabpanel" aria-labelledby="installation-18-04-tab">
+          <h2>Ubuntu 18.04 LTS &mdash; Bionic Beaver</h2>
+          <p>Desktop:</p>
+          <pre><code>sudo apt-get install --install-recommends linux-generic-hwe-18.04 xserver-xorg-hwe-18.04</code></pre>
+          <p>Server:</p>
+          <pre><code>sudo apt-get install --install-recommends linux-generic-hwe-18.04</code></pre>
+          <p>The 18.04.2 and newer point releases will ship with an updated kernel and X stack by default for the desktop. Server installations will default to the GA kernel and provide the enablement kernel as optional.</p>
+          <p>The 18.04 HWE Stacks will follow the Rolling Update Model.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Check your support status</h2>
+      <p>To determine if your install is still supported, use this terminal command:</p>
+      <p>
+        <code>hwe-support-status --verbose</code>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Kernel release schedule</h2>
+      <div class="u-hide--small" id="kernel-schedule"></div>
+      <table class="u-hide--medium u-hide--large">
+        <thead>
+          <tr>
+            <td>&nbsp;</td>
+            <th scope="col">Released</th>
+            <th scope="col">End of Life</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><strong>Ubuntu 14.04 LTS (v3.13)</strong></td>
+            <td>Apr 2014</td>
+            <td>Apr 2019</td>
+          </tr>
+          <tr>
+            <td><strong>Ubuntu 16.04 LTS (v4.4)</strong></td>
+            <td>Apr 2016</td>
+            <td>Apr 2021</td>
+          </tr>
+          <tr>
+            <td><strong>Ubuntu 18.04 LTS (v4.15)</strong></td>
+            <td>Apr 2018</td>
+            <td>Apr 2023</td>
+          </tr>
+          <tr>
+            <td>Ubuntu 19.04 (v5.0)</td>
+            <td>Apr 2019</td>
+            <td>Jan 2020</td>
+          </tr>
+          <tr>
+            <td>Ubuntu 19.10 (v5.3)</td>
+            <td>Oct 2019</td>
+            <td>Jul 2020</td>
+          </tr>
+          <tr>
+            <td><strong>Ubuntu 20.04 LTS</strong></td>
+            <td>Apr 2020</td>
+            <td>Apr 2025</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h3>Kernel support schedule</h3>
```

Comment:
As pointed out by Ana:
```suggestion
      <h2>Kernel support schedule</h2>
```